modyn/supervisor/internal/grpc_handler.py

```diff
@@ -349,6 +354,7 @@ def start_training(
             "grad_scaler_configuration": TrainerServerJsonString(value=json.dumps(grad_scaler_config)),
             "epochs_per_trigger": epochs_per_trigger,
             "seed": seed,
+            "tokenizer": PythonString(value=tokenizer) if tokenizer is not None else None,
```

Comment:
Just using PythonString(value=tokenizer) is not working since it's mapped to PythonString(value="") and not filtered in cleaned_kwargs

Reply:
ok, thanks for letting me know!